modules/data.atmosphere/inst/integrationTests/test.download.AmerifluxLBL.R

```diff
@@ -1,19 +1,23 @@
 library(testthat)
+library(mockery)
+library(PEcAn.DB)
+library(PEcAn.logger)
+library(withr)
 
 test_download_AmerifluxLBL <- function(start_date, end_date, sitename, lat.in, lon.in) {
   # putting logger to debug mode
-  PEcAn.logger::logger.setUseConsole(TRUE, FALSE)
```

Comment:
In general I'm surprised by the need to add the library calls. I'm even more surprised that you need to actively remove the namespace info. Did you try a version that added the library calls without deleting the namespace?

Reply:
Yes it does seem kind of strange. But I did try to add library calls and keep the namespaces but that too resulted in a failure while running the tests.

Reply:
I tried out removing just the `PEcAn.DB` namespace and putting that to a library call. This works well.